### ISSUE TITLE
iOS review failure due to private API name usage

### ISSUE THREAD

**Issue by mfarinets** [fusepowered/FuseSDKiOS#37]
File "Fuseboxx/libFuseSDK.a" contains the method or property named "credentials" which is restricted by Apple.

**Details:** 
2. 5 PERFORMANCE: SOFTWARE REQUIREMENTS
Performance - 2.5.1
```
Your app uses or references the following non-public APIs:
1. credentials
The use of non-public APIs is not permitted on the App Store because it can lead to a poor user experience should these APIs change.
Next Steps
Please revise your app to remove any non-public APIs. If you have defined methods in your source code with the same names as the above-mentioned APIs, we suggest altering your method names so that they no longer collide with Apple's private APIs to avoid your application being flagged in future submissions.
Additionally, if you are using third party libraries, please update to the most recent version of those libraries. If you do not have access to the libraries' source, you may be able to search the compiled binary using the "strings" or "otool" command line tools. The "strings" tool can output a list of the methods that the library calls and "otool -ov" will output the Objective-C class structures and their defined methods. These tools can help you narrow down where the problematic code resides. You could also use the "nm" tool to verify if any third-party libraries are calling these APIs.
```

**Comment by kshamim** [comment]
We're aware of this issue and will have a fix out shortly.

**Comment by kshamim** [comment]
Fixed in v2.6.4 of the SDK
